client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/CentralDogmaEndpointGroup.java

```diff
@@ -146,9 +147,10 @@ protected void doCloseAsync(CompletableFuture<?> future) {
 
     @Override
     public String toString() {
-        return toStringHelper()
+        return toStringHelper(this)
```

Comment:
Using this approach as opposed to using the `toString(Consumer<StringBuilder>)` API since using a `StringBuilder` required me to format things manually